engine/apps/mattermost/serializers.py

```diff
@@ -36,19 +36,13 @@ def to_representation(self, instance):
         return ret
 
     def to_internal_value(self, data):
-        team_name = data.get("team_name")
-        channel_name = data.get("channel_name")
+        channel_id = data.get("channel_id")
```

Comment:
Since we are now using the mattermost channel ID (without referencing team in any way), wondering if we should display the channel list showing channel display name and channel ID (maybe fully displaying it?), instead of team ID (which user may never need to know, and I think it could be clearer in the UI what this ID is?)

Reply:
Yup removed the the unique display name logic and displayed the channel id in the UI directly